lib/node_modules/@stdlib/lapack/base/dlae2/README.md

```diff
@@ -0,0 +1,221 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2024 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# dlae2
+
+> Compute the eigenvalues of a 2x2 symmetric matrix.
+
+<section class = "usage">
+
+## Usage
+
+```javascript
+var dlae2 = require( '@stdlib/lapack/base/dlae2' );
+```
+
+#### dlae2( A, B, C, Out )
```

Comment:
```suggestion
#### dlae2( A, B, C, out )
```

You can lowercase `out` here and below. We use `out` elsewhere in the project and it is always lowercased.